roles/openshift_logging/tasks/generate_deploymentconfigs.yaml

```diff
@@ -4,8 +4,8 @@
   vars:
```

Comment:
this file really needs to be deleted, im pretty sure it isn't called at all.... TODO